vm/src/obj/objdict.rs

```diff
@@ -259,9 +180,8 @@ impl PyDictRef {
         }
     }
 
-    fn setitem(self, needle: PyObjectRef, value: PyObjectRef, _vm: &VirtualMachine) {
-        let mut elements = self.entries.borrow_mut();
-        set_item_in_content(&mut elements, &needle, &value)
+    fn setitem(self, needle: PyObjectRef, value: PyObjectRef, vm: &VirtualMachine) {
```

Comment:
I suggest to rename one of the functions `setitem` and `set_item`. The names are too similar.

Reply:
Yeah, I'm hoping they'll have merged by the time I get to end of this refactor.